packages/builder/src/components/integration/RestQueryViewer.svelte

```diff
@@ -196,8 +196,36 @@
     }
   }
 
+  const validateQuery = async () => {
+    const forbiddenBindings = /{{\s?user\s?}}/g
+    const bindingError = new Error(
+      "'user' is a protected binding and cannot be used"
+    )
+
+    if (forbiddenBindings.test(url)) {
+      throw bindingError
+    }
+
+    if (forbiddenBindings.test(query.fields.requestBody ?? "")) {
+      throw bindingError
+    }
+
+    Object.values(requestBindings).forEach(bindingValue => {
```

Comment:
Can we guarantee `requestBindings` is always defined? Just since otherwise this will throw an error and crash.

Reply:
Always defined as far as I can tell